daemon/src/routes_taker.rs

```diff
@@ -195,14 +194,15 @@ pub async fn get_wallet_mnemonic(
     Ok(status::Accepted(Option::from(mnemonic.to_string())))
```

Comment:
Returning the seed words seem unnecessary. That is exactly the information the caller gave us. Just creates confusion on whether or not the client should do something with this.

Reply:
This is the endpoint to backup your wallet. It has to return the seed words.

Reply:
We need this because we automatically generate seed words for the user upon startup and the user has needs away to get them.